common/build-helper/texmf.sh

```diff
@@ -0,0 +1,25 @@
+# Avoid repeating same thing accross a gazillion files
+
+# rsync isn't needed for everything but it's far easier to just put it here
+hostmakedepends+=" unzip xz rsync"
+# python_version isn't needed for everything either
+python_version=3
+
+do_install() {
+	mkdir -p "${PKGDESTDIR}/usr/share/texmf-dist"
+	unzip "${pkgname}-${version}-src.zip"
+	find . -maxdepth 1 -print -name "*.tar.xz" \
+		-exec bsdtar -C "${PKGDESTDIR}/usr/share/texmf-dist" -xf {} \;
+	cd "${PKGDESTDIR}/usr/share/texmf-dist/"
+	if [ -d "texmf-dist" ] ; then
+		rsync -ar texmf-dist/ ./
+		rm -rf texmf-dist/
+	fi
+	rm -f LICENSE*
+	if [ -f "${FILESDIR}/duplicates.txt" ] ; then
```

Comment:
Files will become outdated quite fast. Is it possible to compute list here?

Reply:
Not really, each file needs to be looked at on a case-by-case basis. It's annoying but necessary. I made a script that checks this. Should I include this somewhere in one of the `files/` directories?

Reply:
Yes it is preferred to share tools used to manage templates. Show it, and i'll try to make it part of build.